### ISSUE TITLE
feat: GitHub README Template Generator

### ISSUE THREAD

**Issue by shravan20** [shravan20/github-readme-quotes#237]
**_Description of feature:_**

This is a custom template generator by specifically select colors for each of the components of the Github Readme Quotes like:
- Readme Card
- Border Color
- Font Color
- Font Border color

**Comment by GRenziITDM** [comment]
Hi @shravan20 can you assign this to me? Thank you.

**Comment by shravan20** [comment]
Yes, thank you so much for your constant support and contribution mate ❤️💫

**Comment by shravan20** [comment]
Assigned it to you

**Comment by GRenziITDM** [comment]
Hi @shravan20, I just wanted to update you on this issue. I implemented two select for font and background color. I saw that border automatically sync with bg color. Is this (screenshot) the expected behavior or do you want me to add other things? Let me know and thank you.

![image](https://github.com/shravan20/github-readme-quotes/assets/20328554/f18c3af7-e429-4480-87e8-7eaa14c00161)
![image](https://github.com/shravan20/github-readme-quotes/assets/20328554/dd8bdab2-caa2-4761-b9c9-acbad4f9263c)
